agoric/versions.json

```diff
@@ -15,6 +15,13 @@
       "height": 15008615,
       "proposal": 74,
       "go_version": "1.20.2"
```

Comment:
add `"next_version_name": "agoric-upgrade-16"` under this line.

Reply:
Thanks! I've done so.